functional-tests/tools/server/server.wizard/server.wizard.validation/src/com/liferay/ide/functional/server/tests/ValidationServerTests.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ide.functional.server.tests;
+
+import com.liferay.ide.functional.liferay.ServerTestBase;
+import com.liferay.ide.functional.liferay.support.server.PureTomcat72Support;
+import com.liferay.ide.functional.liferay.support.server.ServerSupport;
+import com.liferay.ide.functional.swtbot.util.StringPool;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * @author Lily Li
+ */
+public class ValidationServerTests extends ServerTestBase {
+
+	@Override
+	public ServerSupport getTestServer() {
+		if ((testServer == null) || !(testServer instanceof PureTomcat72Support)) {
+			testServer = new PureTomcat72Support(bot);
+		}
+
+		return testServer;
+	}
+
+	@Test
+	public void testDefaultServerName() {
+		dialogAction.openPreferencesDialog();
+
+		dialogAction.preferences.openServerRuntimeEnvironmentsTry();
+
+		dialogAction.serverRuntimeEnvironments.openNewRuntimeWizard();
+
+		wizardAction.newRuntime.prepare7();
+
+		wizardAction.next();
+
+		validationAction.assertEquals(LIFERAY_7_X, wizardAction.newRuntime7.getName());
+
+		validationAction.assertEquals(StringPool.BLANK, wizardAction.newRuntime7.getLiferayPortalBundleDirectory());
+
+		validationAction.assertEquals(StringPool.BLANK, wizardAction.newRuntime7.getDetectedPortalBundleType());
+
+		wizardAction.newRuntime7.prepare(testServer.getFullServerDir());
+
+		validationAction.assertEquals(
+			"Liferay Community Edition Portal 7.2.0 CE GA1", wizardAction.newRuntime7.getName());
```

Comment:
I think we can use "Liferay Community Edition Portal 7"  instead

Reply:
can you help try if server.getServerDirName contians "wizardAction.newRuntime7.getName()"? thanks